features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/SessionCallbackCrashScenario.swift

```diff
@@ -27,7 +27,7 @@ class SessionCallbackCrashScenario : Scenario {
 
         // overwrite app ID set in first callback
```

Comment:
I started by noting that this comment was out of date, but then I realised that the change means losing the test that  values overwritten in the second callback take precedent over those set in the first.

Perhaps add setting of the device id here, instead of replacing setting of the app id - then we keep both tests.  And update the comment of course.

Reply:
Updated the test so that it covers this scenario too.